Gemfile

```diff
@@ -8,10 +8,15 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
   gem "simplecov", require: false
+  # Override the coffee-rails version in engine_cart for rails 4.2
+  case ENV['RAILS_VERSION']
+  when /^4.2/
+    gem 'coffee-rails', '~> 4.2.0'
```

Comment:
A separate issue is whether this override is actually needed at all any more, so I did not intend to address that in this PR.